ledger-core/rms/proto_gen.go

```diff
@@ -7,4 +7,4 @@ package rms
 
 // `go:generate` command below is here just for information. It should not be used by go generate.
 // Actual generation is in the Makefile.
-// go:generate protoc -I=. -I=$GOPATH/src --ins_out=./ proto_*.proto
+//go:generate protoc -I=. -I=$GOPATH/src --ins_out=./ proto_*.proto
```

Comment:
either comment above is outdated, or this string has to have space before

Reply:
that's artifact of my local work - pls ignore ;) WIP